language-plutus-core/test/Generators.hs

```diff
@@ -38,7 +38,8 @@ genBuiltin = BuiltinName emptyPosn <$> genBuiltinName
 genConstant :: MonadGen m => m (Constant AlexPosn)
 genConstant = Gen.choice [genInt, genSize, genBS]
     where int' = Gen.integral_ (Range.linear (-10000000) 10000000)
-          size' = Gen.integral_ (Range.linear 1 10)
+          -- always pick a size big enough to hold the integers we generate
+          size' = Gen.integral_ (Range.linear 4 10)
```

Comment:
Do we need this? For serialization we should test everything, not just sensible programs.

Reply:
Like I said, we can't round-trip `BuiltinInt () 1 100000000` through prettyprinting and parsing, because the parser does a bounds check (perhaps it shouldn't).

I think it's reasonable for our generator to generate valid programs.